apps/chatbot/.env.example

```diff
@@ -5,11 +5,16 @@ LOG_LEVEL=DEBUG
 CHB_AWS_ACCESS_KEY_ID=...
 CHB_AWS_SECRET_ACCESS_KEY=...
 CHB_AWS_DEFAULT_REGION=eu-west-3
+CHB_GOOGLE_AWS_DEFAULT_REGION=eu-south-1
+CHB_REDIS_AWS_DEFAULT_REGION=eu-south-1
```

Comment:
Why do we need a separate variable for GOOGLE and REDIS regions?

Shall we use for example CHB_AWS_DEFAULT_REGION=eu-south-1 and CHB_BEDROCK_AWS_REGION=eu-west-3?

Reply:
ok, I can do this change